src/bjointsp/template/template.py

```diff
@@ -49,11 +49,12 @@ def weight(self, src_dr):
         # iterate over components in topological order, determine outgoing dr for each and store it in out_dr
         # first, consider only forward direction then only backward
         # also determine cpu/mem consumption per component and direction and increase total_cpu/mem accordingly
-        total_cpu, total_mem = 0, 0
+        total_cpu, total_mem = 0, 0, 
```

Comment:
Please remove the comma. Probably doesn't break anything, but just to be on the safe side.